content/blog/2021-06-16-june-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,242 @@
+  🧐 Our MLOps philosophy
+  📖 MLOps learning opportunities,
+  💻 R with DVC, 
+  🎥 conference videos from our team members,
+  🚀 info on our growing team, and more!
+picture: 2021-06-16/june21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/june-21-heartbeat/793?
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - MLOps Community
+  - R
+  - PyData SoCal
+---
+
+# From the Community
+
+This month I'm going to take you on a thought provoking journey through some of
+the content from our community.
+
+![So many choices...](https://media.giphy.com/media/Uni2jYCihB3fG/giphy.gif)
+
+## LJ Miranda Triad of order
+
+The MLOps tool landscape can be confusing to say the least.  
+[LJ Miranda](https://twitter.com/ljvmiranda921), in a well written
+[three-part series](https://ljvmiranda921.github.io/notebook/2021/05/10/navigating-the-mlops-landscape/)
+lays out a framework for making sense of this space. The list of tools is not
+exhaustive, but the framework and thought process for evaluating the tools is
+intriguing. Additionally he encourages thinking about the skillset of the
+members of your team within this framework to help you make decisions on the
+right tools. It's not just about the tools, it's about the people!
+
+As you can see DVC makes it into the "Trial" loop, but we think we will be be
+making it into the adoption region in relatively short order. 😉🚀
+
+![](/uploads/images/2021-06-16/LJMiranda.png)_Making sense of the MLOps
+Landscape_
+
+## Found in the MLOps Community
+
+You can find more comments from LJ Miranda and others in response to a
+[great question](https://mlops-community.slack.com/?redir=%2Farchives%2FC015J2Y9RLM%2Fp1622714574054300)
+from André Godinho in the [MLOps Community](https://mlops.community/) Slack (see
+below). If you're into MLOps and you're NOT a apart of the MLOps Community, you
+should be. You can join the
+[MLOPS Slack channel here](https://mlops-community.slack.com/join/shared_invite/zt-o96abp9z-sRYKWb96wGK9vdhUvbSrsQ#/shared-invite/email).
+
+> I have recently came across with DVC by listening to MLOps Coffee Sessions #6
+> with David Aponte and Elle O'Brien (Such an interesting talk! 💯). This tool
+> integrates smoothly with Git, tracks models & datasets, and also has an online
+> UI DVC Studio 🚀. Is there any use case of MLflow that DVC can't handle? I
+> find DVC to give more rise to creativity as it integrates really well with
+> Git. - André Godinho
+
+## Neda Sultova Tutorial
+
+Drilling down to the next level, I give you
+[this tutorial](https://medium.com/geekculture/exploring-dvc-for-machine-learning-pipelines-in-research-part-1-3ebc2ca35a18)
+by [Neda Sultova](https://www.linkedin.com/in/neda-sultova-597a811a8/). Not only
+is it a great tutorial of DVC in and of itself, but Neda started with her use
+case and the framework for deciding on what tools will bridge the gaps and
+increase efficiency for her team at [Helmholtz AI](https://www.helmholtz.ai/).
```

Comment:
Long sentence. And I'm not sure what's being said here (after ", but").

Reply:
Reworking...